canonical-web-and-design/snapcraft.io
# Sort & update publicise languages with italian and portuguese additions

Fixes #2038 

### QA Steps

1. Pull the branch or run the demo service
2. Log in and check any snap you own on its `/publicise` page
3. Check on the Snap Store badges that Italian and Portuguese are available
4. #winning
